javascript/javascript2/week2/movies.js

```diff
@@ -44630,13 +44630,11 @@ console.log(moviesWithKeywords.length);
 
```

Comment:
I would like to write comment before each exercise to make it readable

Reply:
good suggestion